packages/component/src/createLoadable.js

```diff
@@ -343,10 +343,13 @@ function createLoadable({
 
     // In future, preload could use `<link rel="preload">`
     Loadable.preload = props => {
-      ctor.requireAsync(props)
+      cache[getCacheKey()] = ctor.requireAsync(props);
     }
 
-    Loadable.load = props => ctor.requireAsync(props)
+    Loadable.load = props => {
+      cache[getCacheKey()] = ctor.requireAsync(props);
```

Comment:
I think the right way forward is to (re)use ‘ resolveAsync’ here, but it does mean that you are right 👍 , it’s all about caches.